sim/src/main/makefrag/firesim/Makefrag

```diff
@@ -30,22 +30,57 @@ VERILOG := $(GENERATED_DIR)/FPGATop.v
 HEADER  := $(GENERATED_DIR)/$(DESIGN)-const.h
 
 ifdef FIRESIM_STANDALONE
-	firesim_sbt_project := {file:${chipyard_dir}}firechip
+	firesim_sbt_project ?= {file:${chipyard_dir}}firechip
 
 	lookup_scala_srcs = $(shell find -L $(1)/ -name target -prune -o -iname "[!.]*.scala" -print 2> /dev/null)
-	SOURCE_DIRS = $(chipyard_dir)/generators $(firesim_base_dir)
-	SCALA_SOURCES = $(call lookup_scala_srcs,$(SOURCE_DIRS))
+	SOURCE_DIRS ?= $(chipyard_dir)/generators $(firesim_base_dir)
+	SCALA_SOURCES ?= $(call lookup_scala_srcs,$(SOURCE_DIRS))
 else
-	firesim_sbt_project := firechip
+	firesim_sbt_project ?= firechip
 endif
 
-$(FIRRTL_FILE) $(ANNO_FILE): $(SCALA_SOURCES) $(FIRRTL_JAR) $(SCALA_BUILDTOOL_DEPS)
-	mkdir -p $(@D)
-	$(call run_scala_main,$(firesim_sbt_project),chipyard.Generator,\
+BOOTROM_FILES = bootrom.rv32.img bootrom.rv64.img
+CHIPYARD_ROMDIR ?= $(chipyard_dir)/generators/testchipip/bootrom
+# pattern rule to handle copying bootrom images into GENERATED_DIR so that they are available for
+# - distributed elaboration on a build host
+# - dromajo co-sim
+BOOTROM_TARGETS = $(addprefix $(GENERATED_DIR)/, $(BOOTROM_FILES))
+$(BOOTROM_TARGETS): $(GENERATED_DIR)/bootrom.%.img: $(CHIPYARD_ROMDIR)/bootrom.%.img | $(GENERATED_DIR)
```

Comment:
Im not opposed to this... but if we want to dedup even more we could make the `none` Simulator option in `GenerateSimFiles` (in Chipyard) just copy the bootrom... I think that should work with the FPGA prototyping stuff.